extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -140,5 +142,12 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		registeredCallbacks[id] = callback;
 	};
 
+	GoogleTag.prototype.onAdLoad = function (slotName, element, gptEvent, onAdLoadCallback) {
```

Comment:
This is a new functionality in our `GoogleTag` wrapper I wonder how much it affects ads' load time. What do you think about checking it with some benchmarking?